style.css

```diff
@@ -533,28 +563,80 @@ body {
     margin-top: 0;
   }
 
+  .project-content-2 {
+    width: 50%;
+    display: flex;
+    flex-direction: column;
+    justify-content: space-between;
+    margin-top: 0;
+  }
+
   .desktop-image-div-2 {
-    float: right;
     display: flex;
     margin-right: 2.5%;
     width: 50%;
     justify-content: space-between;
     flex-direction: column;
     margin-bottom: 0;
+    order: 2;
   }
 
-  .project-content-2 {
-    float: none;
+  .project-content-4 {
     width: 50%;
     display: flex;
     flex-direction: column;
     justify-content: space-between;
     margin-top: 0;
   }
 
+  #skills-list {
+    list-style: none;
+    padding: 8%;
+  }
+
+  #skills-list li {
+    margin-bottom: 15%;
+    height: fit-content;
+  }
+
+  .skills-dropdown {
+    display: flex;
+    height: fit-content;
+    flex-direction: row;
+  }
+
+  .skills-dropdown-1 {
+    background-color: #f7f7f8;
+    margin: 2.9% 9% 10% -2.9%;
+    border-radius: 12px;
+    padding: 14%;
+    padding-right: 30px;
+  }
+
+  .skill-name {
+    position: absolute;
+    margin-left: -12%;
+    margin-top: 13%;
+    color: #253858;
+    font-style: normal;
+    font-weight: 500;
+    font-size: 17px;
+    line-height: 20px;
+  }
```

Comment:
- [OPTIONAL] You have done a fantastic job with your work :+1:. However, please take a moment to check each language that you added to your languages section. Ensure that the text (JavaScript, HTML and CSS) is inside the same background as shown in the Figma template. This is crucial to maintain a consistent design and ensure that the page looks polished and professional. By matching the text background to the template, you can provide users with a more cohesive and visually appealing experience.

**Your design**

![Screenshot from 2023-04-01 10-48-56](https://user-images.githubusercontent.com/105117832/229301066-add97146-fa29-498e-bcbf-c17d41f23f66.png)

**Figma**

![Screenshot from 2023-04-01 10-49-59](https://user-images.githubusercontent.com/105117832/229301098-1186c6af-c0dd-4e8b-abf8-215b6583dc11.png)

Reply:
Thank you very much @IngDavidTM  for your output. I am integrating these changes.